Makefile

```diff
@@ -71,3 +72,24 @@ check:
 		   test/integration/lib.sh test/integration/rebase-patches	\
 		   test/integration/test-vagrant				\
 		   test/integration/vm-integration-run
+
+help:
+	@echo "kpatch Makefile"
+	@echo
+	@echo "Targets:"
+	@echo "    make dependencies                 install build dependencies [1]"
+	@echo "    make all                          build entire project"
+	@echo "    make install                      install programs to system [1]"
+	@echo "    make uninstall                    remove programs from system [1]"
+	@echo "    make clean                        clean build files"
+	@echo
+	@echo "Test targets:"
+	@echo "    make check                        run static code analyzers"
+	@echo "    make integration                  build and run integration tests [2]"
+	@echo "    make integration-slow             build and run integration tests [2]"
+	@echo "    make integration-quick            build and run integration tests [2]"
+	@echo "    make unit                         run unit tests"
+	@echo
+	@echo "[1] requires admin privileges"
+	@echo "[2] installs test kpatch kernel modules, run at your own risk"
+	@echo
```

Comment:
(I skipped the vagrant targets since we don't document them on any markdown page.)